<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgCaptureWidget</class>
 <widget class="QWidget" name="ImgCaptureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1808</width>
    <height>906</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_13">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_img1">
         <property name="styleSheet">
          <string notr="true">font-family: '微软雅黑';
font-size: 16px;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>图片显示：第一张</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_img1">
         <property name="minimumSize">
          <size>
           <width>601</width>
           <height>851</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11" stretch="0,0">
       <item>
        <widget class="QLabel" name="label_img2">
         <property name="styleSheet">
          <string notr="true">font-family: '微软雅黑';
font-size: 16px;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>图片显示：第二张</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_img2">
         <property name="minimumSize">
          <size>
           <width>601</width>
           <height>851</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_arduino">
             <property name="styleSheet">
              <string notr="true">font-family: 'Times New Roman';
font-size: 12px;
font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Arduino串口设置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_arduino">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-family: 'Times New Roman';
font-size: 16px;
font-weight: bold;</string>
             </property>
             <property name="currentText">
              <string>com7</string>
             </property>
             <item>
              <property name="text">
               <string>com7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com8</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_scanner">
             <property name="styleSheet">
              <string notr="true">font-family: 'Times New Roman';
font-size: 12px;
font-weight: bold;</string>
             </property>
             <property name="text">
              <string>扫描器串口设置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_scanner">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-family: 'Times New Roman';
font-size: 16px;
font-weight: bold;</string>
             </property>
             <item>
              <property name="text">
               <string>com7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com8</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_serial_connect">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);
font-family: '微软雅黑';
font-size: 16px;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>确认串口连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btn_forward">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 85, 255);color: white;
font-family: 'Times New Roman';
font-size: 16px;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>前进</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_back">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-family: 'Times New Roman';
font-size: 16px;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>后退</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_stop">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);
font-family: 'Times New Roman';
font-size: 16px;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_log">
           <property name="text">
            <string>当前处理信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="tb_log">
           <property name="minimumSize">
            <size>
             <width>560</width>
             <height>660</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
